Algorithm/QCAlgorithm.cs

```diff
@@ -314,48 +310,27 @@ public virtual void OnTick(Dictionary<string, List<Tick>> data)
             //throw new NotImplementedException("OnTick has been made obsolete. Please use OnData(Ticks data) instead.");
         }
 
-        // <summary>
-        // Event - v2.0 TRADEBAR EVENT HANDLER: (Pattern) Basic template for user to override when requesting tradebar data.
-        // </summary>
-        // <param name="data"></param>
-        //public void OnData(TradeBars data)
-        //{
-        //
-        //}
-
-        // <summary>
-        // Event - v2.0 TICK EVENT HANDLER: (Pattern) Basic template for user to override when requesting tick data.
```

Comment:
Agree, please leave in for doc. Its helpful for others trying to understand how the event system works.